package/yast2-ntp-client.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Fri Jun 30 09:06:48 UTC 2017 - jlopez@suse.com
+
+- AutoYaST: Fix crash when sanitizing values
```

Comment:
we could explain exactly which values, and at least modify this profile adding the 

```<initial_sync config:type="boolean">true</initial_sync>```

to the first peer in:

test/fixtures/autoyast/autoinst.xml